kernel/src/main/scala/org/apache/toree/boot/CommandLineOptions.scala

```diff
@@ -71,6 +71,14 @@ class CommandLineOptions(args: Seq[String]) {
     parser.accepts("magic-url", "path to a magic jar")
       .withRequiredArg().ofType(classOf[String])
 
+  private val _default_repositories = parser.accepts(
+    "default_repositories", "comma seperated list of additional repositories to resolve"
```

Comment:
It's a little confusing, but you should use hyphens instead of underscores for the CLI argument names. E.g. `default-repositories`. Down below, you are mapping the CLI value to the config property, which will have an underscore (so you're okay there). I believe this is what is causing our existing tests to fail in your PR.